integrations/azure-devops/.port/resources/port-app-config.yaml

```
@@ -35,3 +39,20 @@ resources:
           blueprint: '"service"'
           properties:
             workItemLinking: '.isEnabled and .isBlocking'
+
+  - kind: project
+    selector:
+      query: 'true'
+      defaultTeam: "true"
+    port:
+      entity:
+        mappings:
+          identifier: '.id'
+          blueprint: '"project"'
+          "title": .name
```

```suggestion
          title: '.name'
```
Make sure to change every other property to the convention